travis-ci.md

```diff
@@ -0,0 +1,16 @@
+---
+title:  Travis CI
+date:   2017-03-16 19:26:48
+categories: Development
+tags: [Development]
+permalink: /travis-ci/
+keywords: travis, continuous integration, development, environment, install, automated testing
+---
+Very simple. Register, sync your GitHub repo, commit and push.  
+
+1. Sign in to [https://travis-ci.org/](https://travis-ci.org/)
+
+2. Go to [https://travis-ci.org/profile/](https://travis-ci.org/profile/) and enable _username_/openclassifieds2. 
```

Comment:
Depends on what is the name of the forked repo. I thought we all still have openclassifieds2, I am checking how to make it as a variable.